### PR TITLE
Add installation guide

### PR DESCRIPTION
**Pull request by shelld3v**
Many people are installing this by: `git clone https://github.com/pwntester/ysoserial.net`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,6 +21,14 @@ This software has been created purely for the purposes of academic research and
 
 This software is a personal project and not related with any companies, including Project owner and contributors employers.
 
+## Installation
+You can install the software from this page: https://github.com/pwntester/ysoserial.net/releases
```
Comment (pwntester) [inline README.md:25]:
```suggestion
You can install the latest version and previous releases of YSoSerial.NET from [the releases page](https://github.com/pwntester/ysoserial.net/releases)
```

File: README.md [hunk 2]
```diff
@@ -21,6 +21,14 @@ This software has been created purely for the purposes of academic research and
...
+
+- Release 1.34: [Download](https://github.com/pwntester/ysoserial.net/releases/download/v1.34/ysoserial-1.34.zip) (latest version)
+- Release 1.33: [Download](https://github.com/pwntester/ysoserial.net/releases/download/v1.33/ysoserial-1.33.zip)
+- Release 1.32: [Download](https://github.com/pwntester/ysoserial.net/releases/download/v1.32/ysoserial-1.32.zip)
+- Release 1.31: [Download](https://github.com/pwntester/ysoserial.net/archive/v1.31.zip)
+
```
Comment (pwntester) [inline README.md:31]:
```suggestion
```

**Review by pwntester — CHANGES_REQUESTED** [review]
Thanks for the typo fix! I dont think we need individual download links for old versions though

**Comment by shelld3v** [comment]
Hi, I think that many people are so familiar with `git clone`, so they may don't know what to do on the releases page (there are notes, source codes, ...), I used to be like this. So it's maybe better to include the download links in the README page!

File: README.md [hunk 3]
```diff
@@ -21,6 +21,14 @@ This software has been created purely for the purposes of academic research and
...
 This software is a personal project and not related with any companies, including Project owner and contributors employers.
 
+## Installation
+You can install the latest version and previous releases of YSoSerial.NET from [the releases page](https://github.com/pwntester/ysoserial.net/releases)
+
+- Release 1.34: [Download](https://github.com/pwntester/ysoserial.net/releases/download/v1.34/ysoserial-1.34.zip) (latest version)
+- Release 1.33: [Download](https://github.com/pwntester/ysoserial.net/releases/download/v1.33/ysoserial-1.33.zip)
+- Release 1.32: [Download](https://github.com/pwntester/ysoserial.net/releases/download/v1.32/ysoserial-1.32.zip)
+- Release 1.31: [Download](https://github.com/pwntester/ysoserial.net/archive/v1.31.zip)
+
```
Comment (pwntester) [inline README.md:31]:
```suggestion
- Release 1.34 (latest version): [Download](https://github.com/pwntester/ysoserial.net/releases/download/v1.34/ysoserial-1.34.zip)

```

**Review by pwntester — CHANGES_REQUESTED** [review]

**Review by pwntester — APPROVED** [review]

**Comment by pwntester** [comment]
thanks!

**Pull request merged**